src/MoBi.Engine/Sbml/EventImporter.cs

```diff
@@ -90,6 +93,15 @@ internal void CreateEvent(Event sbmlEvent)
          EventGroupBuilder.Add(EventBuilder);
       }
 
+      private void CreateEmptyEvent()
+      {
+         EventGroupBuilder.Add(
+            new EventBuilder()
+               .WithId(SBMLConstants.SBML_EVENTBUILDER)
+               .WithName(SBMLConstants.SBML_DEFAULTEVENTNAME)
+         );
+      }
```

Comment:
this looks suspicios for me. Why not creating the new event builder via `Context.Create()` ?
Especially if we import >1 sbml simulations - it might create several event builders with the same Id (or even crash?).
@msevestre Maybe you can comment on this.

Reply:
@msevestre and @Yuri05, I am using the same code style. I would say if we need to change this, there are several places where we need to do the same. Do notice that at this point, the object is not yet added to the project. It will later be added when the method AddToProject is invoked (which also updates the history). Could this be the cause?

Reply:
`CreateEmptyEvent()` is only called if no Events-BB is created during import, so there should be no issues about IDs.

Reply:
If I import >1 SBML simulations into the same project - no issues as well?

Reply:
Not possible - "import sbml" always starts an new project. We might want to change this later but now it is like selecting "Open simulation"